templates/url_redirect/index.html.twig

```diff
@@ -3,26 +3,33 @@
 {% block title %}{% trans %}Redirected Urls {% endtrans %}{% endblock %}
 
 {% block body %}
-    <h1>{% trans %}All your redirected urls{% endtrans %}</h1>
+    <h1>{% trans %}All your generated urls{% endtrans %}</h1>
 
     <table class="table">
         <thead>
             <tr>
                 <th>Id</th>
-                <th>{% trans %}Your Url {% endtrans %}</th>
+                <th>{% trans %}Your Url {% endtrans %}
+                <th>{% trans %}Description {% endtrans %}</th>
                 <th>{% trans %}Generated Url{% endtrans %}</th>
+                <th>{% trans %}Actions{% endtrans %}</th>
             </tr>
         </thead>
         <tbody>
-        {% for url_redirect in url_redirect %}
+        {% for redirects in url_redirect %}
```

Comment:
{% for redirects in url_redirect %} lepszą nazwą byłoby {% for **redirect** in **url_redirects** %} (tylko pamiętaj, żeby to poprawić też w kontrolerze.